puppet/modules/freight/manifests/init.pp

```diff
@@ -1,14 +1,9 @@
 # @summary install freight
 class freight {
-  if $facts['os']['family'] == 'Debian' {
-    apt::source { 'freight':
-      location => 'http://build.openvpn.net/debian/freight_team',
-      repos    => 'main',
-      key      => {
-        id     => '30EBF4E73CCE63EEE124DD278E6DA8B4E158C569',
-        source => 'https://swupdate.openvpn.net/repos/repo-public.gpg',
-      },
-      before   => Package['freight'],
+  if $facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] == '9' {
+    class {'epel':
+      epel_testing_enabled => true,
```

Comment:
Note to self: once https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2024-87076f97c3 is in then we can drop this line.

Reply:
It should go stable in 2 days. Given all the releases that are ongoing I'd prefer to wait for that and avoid introducing this line.

Reply:
absolutely. I just have it here so that my local env works.